website_event_require_login/controllers/main.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import http
+from odoo.addons.website_event.controllers.main import WebsiteEventController
+
+
+class RequireLoginToRegister(WebsiteEventController):
+
+    @http.route()
```

Comment:
Have you tested just overriding this and putting `@http.route(auth="user")`? If I don't remember bad, doing that autoredirects to `/web/login` for unauthenticated users. :thinking:

Reply:
Sure, but we don't want to restrict it to all users